src/components/landing-page/call-to-action/CallToAction.jsx

```
@@ -1,45 +1,54 @@
-import {
-  CoursesCard,
-  SkillCards,
-  WhatOurUsersSay,
-} from "./Cards";
-import {
-  courses,
-  skillContent,
-  whatOurUsersSay,
-} from "./Content";
+import React, { lazy, Suspense } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)